### PR TITLE
[DOCFIX] Add new Deploying Alluxio section to docs

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

**Comment by RongGu** [comment]
LGTM, thanks for the improvement.

**Comment by apc999** [comment]
LGTM

File: docs/_config.yml [hunk 1]
```diff
@@ -40,6 +40,10 @@ categoryInfo:
     en: User Guides
     cn: 用户指南
     pt: Guias do Usuário
+  Deploying Alluxio:
+    en: Deploying Alluxio
+    cn: Deploying Alluxio
+    pt: Deploying Alluxio
```
Comment (waltonryan) [inline docs/_config.yml]: @gpang could anyone provide the Chinese and Portuguese translation here?

File: docs/pt/Running-Alluxio-on-a-Cluster.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 layout: global
 title: Alluxio Standalone em um Cluster
 nickname: Alluxio Standalone em um Cluster
```
Comment (waltonryan) [inline docs/pt/Running-Alluxio-on-a-Cluster.md]: Is `em um` in the title a typo?

**Comment by waltonryan** [comment]
LGTM with minor comments.

File: docs/_config.yml (hunk 1 above)
Comment (RongGu) [inline docs/_config.yml]: Hey @waltonryan , we are doing the Chinese translation.

Comment (waltonryan) [inline docs/_config.yml]: Sounds good. Thanks @RongGu.

**Pull request merged**